core/middleware.py

```diff
@@ -64,7 +64,10 @@ def __call__(self, request):
                     )
 
                 # Create ActivityStream object
-                (as_user, _,) = ActivityStreamStaffSSOUser.objects.update_or_create(
+                (
+                    as_user,
+                    _,
+                ) = ActivityStreamStaffSSOUser.objects.update_or_create(
```

Comment:
I think we can just drop these brackets like so:
```suggestion
                as_user, _ = ActivityStreamStaffSSOUser.objects.update_or_create(
```